### ISSUE TITLE
change default value of `flatten` to False when calling the Transform

### ISSUE THREAD

**Issue by eramdiaz** [lf1-io/padl#24]
I would change the default value of the `flatten` argument of the `__call__` method on `Transform` to `False`, since it is usually more useful to inspect the output of your model by batches, specially when you are looping and computing a loss function over them. I think that the `flatten=False` behaviour feels much more natural and less confusing for the starting user who wants to make his first training loop using `aleph` (also we don't have clear if we are releasing the trainers).
